File: SimpleInterest.java
import java.util.Scanner;
class SimpleInterest {
    public static void main(String[] args) {
        Scanner read = new Scanner(System.in);
        System.out.print("Enter the time period:");
        int t = read.nextInt();
        System.out.print("Enter the Principal amount:");
        int a = read.nextInt();
        System.out.print("Enter the Annual Rate:");
        int r = read.nextInt();
        int s = (t*a*r)/100;
        System.out.println("The Simple Intrest is:"+s);
        System.out.print("The total amount is:"+(s+a));
    }
}

File: SimpleInterest.c
#include<stdio.h>
int main(){
    int a,t,r;
    printf("This program is used to find Simple interest\n");
    printf("Enter the time period(in years):");
    scanf("%d",&t);
    printf("Enter the amount:");
    scanf("%d",&a);
    printf("Enter the annual rate:");
    scanf("%d",&r);
    int s;
    s = (t*r*a)/100;
    printf("The annual interest amount is: %d\n", s);
    printf("The total amount is:%d", s+a);
    return 0;
}
